### ISSUE TITLE
Add more hashing algorithms other than MD5

### ISSUE THREAD

**Issue by TheDGOfficial** [LifeMC/LifeSkript#39]
**Is your feature request related to a problem? Please describe.**
We can't use algorithms other than MD5 in hash expression.

**Describe the solution you'd like**
Adding SHA1 and possibly more hashing algorithms will be great.

**Describe alternatives you've considered**
N/A

**Additional information**
Resolves recently added todo comment:
https://github.com/LifeMC/LifeSkript/blob/75de3704da6749f196642cc5bd9f6cd8b30ac7d9/src/main/java/ch/njol/skript/expressions/ExprHash.java#L50

**Comment by TheDGOfficial** [comment]
I mistyped, no SHA1, it's also weak. probably SHA256 or SHA512 (with or without a salt) should fit requirements for a basic auth script. I think no one is using a script for authenication, but supporting only MD5 is a vulnerability and this should be fixed by supporting more algorithms.

A small research proves that MD5 and SHA1 is also dead. SHA256 and SHA512 are living last days. The best choice seems to be Argon2. I found a java implementation of it, but i don't know.

I should include this in 2.2.16 for this time. No more delaying. But at least i don't include this on beta release, yet. 2.2.16-beta1 and beta2 should out soon. After that i plan working on real 2.2.16 and create beta releases only if a critical bug is fixed.

If no bugs are found in beta releases, 2.2.16 will come soon. It corrects and adjusts many things, so we can focus on other things.
